.github/workflows/docs_workflow.yml

```diff
@@ -0,0 +1,83 @@
+name: CI/CD Release Workflow
+
+on:
+  push:
+    branches: [main]
+
+  release:
+    types: [published]
+
+jobs:
+  publish-docs:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+
```

Comment:
@xylar I removed from here a section in the polaris docs workflow:

```yaml
      - name: Cache Conda
        uses: actions/cache@v4
        env:
          # Increase this value to reset cache if deploy/conda-dev-spec.template has not changed in the workflow
          CACHE_NUMBER: 0
        with:
          path: ~/conda_pkgs_dir
          key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
            hashFiles('configure_polaris_envs.py,deploy/*') }}
```
I just didn't understand what the `hashFiles` function was being used for and what to provide it instead of the configure file.

Reply:
@altheaden, yep, it's a lot to follow. The idea is that the CI can use packages it cached from the last time it ran unless specific files have changed. I will have to think about what the equivalent would be for mache.

Reply:
Maybe just spec-file.txt or whatever it's called.

Reply:
BTW, I really appreciate that you ask good questions when you're not sure about something rather than making a blind guess.

Reply:
OK, I'll try using the spec file and see how far I get.